extensions/adobe/experience/adcloud/adconversions.schema.json

```diff
@@ -0,0 +1,225 @@
+        },
+        "xdm:accountId": {
+          "title": "Account Identifier",
+          "type": "string",
+          "description": "Account ID for the search advertising platform campaign that initiated the conversion event"
+        },
+        "xdm:campaignId": {
+          "title": "Campaign Identifier",
+          "type": "string",
+          "description": "Campaign ID on the advertising platform"
+        },
+        "xdm:placementId": {
+          "title": "Placement Identifier",
+          "type": "string",
+          "description": "Placement ID on the display advertising platform. It will be populated only for non-search advertising related campaigns"
+        },
+        "xdm:adgroupId": {
+          "title": "Ad Group Identifier",
+          "type": "string",
+          "description": "Ad Group ID on the search advertising platform"
+        },
+        "xdm:adId": {
+          "title": "Ad Identifier",
+          "type": "string",
+          "description": "Ad ID on the advertising platform"
+        },
+        "xdm:term": {
+          "title": "Search Term",
+          "type": "string",
+          "description": "Search Term that contributed to the conversion"
+        },
+        "xdm:termId": {
+          "title": "Search Term Identifier",
+          "type": "string",
+          "description": "Search Term ID on the search advertising platform"
+        },
+        "xdm:matchType": {
+          "title": "Term Match Type",
+          "type": "string",
+          "description": "Match Type associated to the Search Term being targeted for the Ad",
+          "enum": [
+            "Broad",
+            "Phrase",
+            "Exact",
+            "Advanced",
+            "Standard",
+            "Website",
+            "Product Target",
+            "Product Group",
+            "unknown"
+          ],
+          "meta:enum": {
+            "Broad": "Broad",
+            "Phrase": "Phrase",
+            "Exact": "Exact",
+            "Advanced": "Advanced",
+            "Standard": "Standard",
+            "Website": "Website",
+            "Product Target": "Product Target",
+            "Product Group": "Product Group",
+            "null": "Unknown"
+          }
+        },
+        "xdm:lastClickUid": {
+          "title": "Last Click UID",
+          "type": "string",
+          "description": "Unique Identifier of the Last Click Event"
+        },
+        "xdm:transactionId": {
+          "title": "Transaction Identifier",
+          "type": "string",
+          "description": "Click Transaction Identifier of the Event"
+        },
+        "xdm:geoipLocationId": {
+          "title": "Geo IP Location Identifier",
+          "type": "number",
+          "description": "A numeric identifier that is obtained from Maxmind’s GeoIP package that represents the Geo"
+        },
+        "xdm:domainPlacement": {
+          "title": "Domain Placement",
+          "type": "string",
+          "description": "Placement Domain"
```

Comment:
If this means the URL of the Publisher, can you please add that to the `title` and `description` ?

Reply:
Thanks. There is another field called referrer URL, hope they have different meanings.